data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -302,16 +237,16 @@ else if (nodeSelExpr.getChild(0).getType() == HiveParser.TOK_ALLCOLREF)
 	private void handleTokTabOrColOrElse(ASTNode nodeSelExpr) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "handleTokTabOrColOrElse" is 14 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)